### PR TITLE
[Toolbox] Add more commands

### PR DESCRIPTION
**Pull request by Yakitrak**
## Description

This extension is really simple and useful, just had a few commands which I wanted and hopefully will be useful for others too:
- JavaScript Object to JSON
- JSON to JavaScript Object
- Escape JSON
- Unescape JSON

## Screencast

toolbox 2023-12-05 at 19 24 00
toolbox 2023-12-05 at 19 22 36

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @Yakitrak 🔥 

I'm awaiting review from @kkange before we can merge, let's give it a day 🙂

File: extensions/toolbox/src/script/format/json.ts
Comment (kkange) [reply extensions/toolbox/src/script/format/json.ts:58]: Thanks for the PR
I think we'll just change the wording of the example 

👍

**Comment by pernielsentikaer** [comment]
Could you look into this @Yakitrak

File: extensions/toolbox/src/script/format/json.ts
Comment (Yakitrak) [reply extensions/toolbox/src/script/format/json.ts]: Thank you for the feedback - I've updated the example.

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**
